pkg/cdc/generate_config.go

```diff
@@ -90,17 +91,46 @@ func updateAdditionalJVMOpts(optsSlice []string, CDCConfig *cassdcapi.CDCConfigu
 			optsSlice = append(optsSlice, nameTag+"="+fmt.Sprintf("%s", reflectedValue))
 		}
 	}
+	// We need to add these two elements to the slice first, because the management agent must start before the CDC agent.
+	// If MCAC is disabled, we need to avoid adding it to startup.
```

Comment:
Same here, this refers to MCAC but the next line deals with CDC. Shouldn't this be merged with line 97 rather?

Reply:
Ditto here.